tests/pipelines/test_pipelines_audio_classification.py

```diff
@@ -109,6 +111,10 @@ def test_small_model_pt(self):
         output = audio_classifier(audio_dict, top_k=4)
         self.assertIn(nested_simplify(output, decimals=4), [EXPECTED_OUTPUT, EXPECTED_OUTPUT_PT_2])
 
+    def test_hub_spec_compliance(self):
+        compare_pipeline_args_to_hub_spec(AudioClassificationPipeline, AudioClassificationInput)
+        raise ValueError("Checking if test actually running in CI")
```

Comment:
We could also test the output I believe! For example with: https://github.com/huggingface/huggingface_hub/blob/f984cdcf98d8ddc8b95b603779d206fbff9d76cf/src/huggingface_hub/inference/_generated/types/audio_classification.py#L39

Reply:
Done! I had to put the output tests in `run_task_test`, because I can't get pipeline outputs anywhere else.